src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,64 @@
+import testRule from "../../../testUtils/stylelint-test-rule-tape"
+import rule, { ruleName, messages } from ".."
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "@import 'x.css';\na {}",
+  }, {
+    code: "@charset 'UTF-8';\na {}",
+  }, {
+    code: "@charset 'UTF-8';\n@import 'x.css'",
+  }, {
+    code: "@charset 'UTF-8';\n@import 'x.css'\na {}",
+  }, {
+    code: "@namespace url(XML-namespace-URL);\na {}",
+  }, {
+    code: "@import 'x.css'); /* comment */\n",
+  }, {
+    code: "@import 'x.css');/* comment */\n",
+  }, {
+    code: "@import 'x.css');   /* comment */\n",
+  }, {
+    code: "@import 'x.css');\t/* comment */\n",
+  }, {
+    code: "@import 'x.css'); \t/* comment */\n",
+  }, {
+    code: "@import 'x.css';\r\n",
+    description: "windows",
+  }, {
+    code: "@import 'x.css'; /* comment */\r\n",
+    description: "windows",
+  }, {
+    code: "@import 'x.css';",
+    description: "single blockless rule",
+  }, {
+    code: "a{\n@extend .b;\n@extend .c\n}",
+    description: "non-standard nested rule",
+  } ],
```

Comment:
I just want to see one more positive test for this common use case: blockless at-rule followed by rule.